DevEdu.API/DevEdu.Db/StoredProcedures/Task_Student/Task_Student_SelectAll.sql

```diff
@@ -7,6 +7,7 @@ BEGIN
 		tstud.StudentId,
 		tstud.Answer,
 		tstud.CompletedDate,
+		tstud.IsDeleted,
```

Comment:
we don't need to return deleted entities
also, we don't need this stored procedure at all